css/src/_m-message.scss

```diff
@@ -0,0 +1,72 @@
+.m-messages {
+    padding: 0;
+    font-style: italic;
+    list-style: none;
+
+    &__item {
+        --message-txt: #da394c;
+        --message-bg: rgba(238, 162, 171, 0.25);
+        position: relative;
+        width: fit-content;
+        margin-inline-end: 11px;
+        margin-block-end: 20px;
+        padding: 10px 15px;
+
```

Comment:
Remove this empty line.

Reply:
Removed the empty line in the 14th line.